### ISSUE TITLE
Process 'command 'python3'' finished Issue

### ISSUE THREAD

**Issue by msh-nayan** [chaquo/chaquopy#562]
Hello,
Greetings. 

#### Chaquopy version

9.1.0 (paid version)

#### Relevant parts of your build.gradle file

(1) root label build.gradle file

```
dependencies {
        classpath "com.android.tools.build:gradle:4.1.2"
        classpath "com.chaquo.python:gradle:9.1.0"
}
```

(2) app label build.gradle file 
```
plugins {
    id 'com.android.application'
    id 'com.chaquo.python'
}
```

```
sourceSets {
            main {
                python {
                    srcDirs = ["src/main/python"]
                }
            }
        }
        python {
            buildPython "/usr/bin/python3"
            buildPython "python3"
            pip {
                install "numpy"
                install "pandas"
                install "scikit-learn"
                install "tensorflow"
                install "h5py"
            }
        }
        ndk {
            abiFilters "armeabi-v7a", "arm64-v8a", "x86", "x86_64"
        }
```

#### Describe your issue

Everything was going smoothly. Last evening (2021-11-16), I was able to build & run the project. Just next morning, today (2021-11-17), when I build & run the project, I got the following error.

```
Executing tasks: [:app:assembleBjitDebug] in project /home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid


> Configure project :app
Warning: This version of Chaquopy has not been tested with Android Gradle plugin versions beyond 4.1.1. If you experience problems, please edit the version of com.android.tools.build:gradle in your top-level build.gradle file. See https://chaquo.com/chaquopy/doc/current/versions.html.

> Task :app:preBuild UP-TO-DATE
> Task :app:preBjitDebugBuild UP-TO-DATE
> Task :app:compileBjitDebugAidl NO-SOURCE
> Task :app:compileBjitDebugRenderscript NO-SOURCE
> Task :app:generateBjitDebugBuildConfig UP-TO-DATE
> Task :app:extractPythonBuildPackages UP-TO-DATE

> Task :app:generateBjitDebugPythonRequirements
Chaquopy: Installing for arm64-v8a
Looking in indexes: https://pypi.org/simple, https://chaquo.com/pypi-7.0
Collecting numpy
  Using cached https://chaquo.com/pypi-7.0/numpy/numpy-1.21.4-1-cp38-cp38-android_21_arm64_v8a.whl
Collecting pandas
  Using version 1.3.2 (newest version is 1.3.4, but Chaquopy prefers native wheels)
  Using cached https://chaquo.com/pypi-7.0/pandas/pandas-1.3.2-0-cp38-cp38-android_21_arm64_v8a.whl
Collecting scikit-learn
  Using version 0.24.1 (newest version is 1.0.1, but Chaquopy prefers native wheels)
  Using cached https://chaquo.com/pypi-7.0/scikit-learn/scikit_learn-0.24.1-0-cp38-cp38-android_21_arm64_v8a.whl
Collecting tensorflow
  Using cached https://chaquo.com/pypi-7.0/tensorflow/tensorflow-2.1.0-9-cp38-cp38-android_21_arm64_v8a.whl
Collecting h5py
  Using version 2.10.0 (newest version is 3.6.0, but Chaquopy prefers native wheels)
  Using cached https://chaquo.com/pypi-7.0/h5py/h5py-2.10.0-1-cp38-cp38-android_21_arm64_v8a.whl
Collecting chaquopy-libgfortran>=4.9 (from numpy)
  Using cached https://chaquo.com/pypi-7.0/chaquopy-libgfortran/chaquopy_libgfortran-4.9-0-py3-none-android_21_arm64_v8a.whl
Collecting chaquopy-openblas>=0.2.20 (from numpy)
  Using cached https://chaquo.com/pypi-7.0/chaquopy-openblas/chaquopy_openblas-0.2.20-5-py3-none-android_21_arm64_v8a.whl
Collecting pytz>=2017.3 (from pandas)
  Using cached https://files.pythonhosted.org/packages/d3/e3/d9f046b5d1c94a3aeab15f1f867aa414f8ee9d196fae6865f1d6a0ee1a0b/pytz-2021.3-py2.py3-none-any.whl
Collecting python-dateutil>=2.7.3 (from pandas)
  Using cached https://files.pythonhosted.org/packages/36/7a/87837f39d0296e723bb9b62bbb257d0355c7f6128853c78955f57342a56d/python_dateutil-2.8.2-py2.py3-none-any.whl
Collecting chaquopy-libcxx>=10000 (from pandas)
  Using cached https://chaquo.com/pypi-7.0/chaquopy-libcxx/chaquopy_libcxx-10000-0-py3-none-android_21_arm64_v8a.whl
Collecting threadpoolctl>=2.0.0 (from scikit-learn)
  Using cached https://files.pythonhosted.org/packages/ff/fe/8aaca2a0db7fd80f0b2cf8a16a034d3eea8102d58ff9331d2aaf1f06766a/threadpoolctl-3.0.0-py3-none-any.whl
Collecting joblib>=0.11 (from scikit-learn)
  Using cached https://files.pythonhosted.org/packages/3e/d5/0163eb0cfa0b673aa4fe1cd3ea9d8a81ea0f32e50807b0c295871e4aab2e/joblib-1.1.0-py2.py3-none-any.whl
Collecting scipy>=0.19.1 (from scikit-learn)
  Using version 1.4.1 (newest version is 1.7.2, but Chaquopy prefers native wheels)
  Using cached https://chaquo.com/pypi-7.0/scipy/scipy-1.4.1-0-cp38-cp38-android_21_arm64_v8a.whl
Collecting gast==0.2.2 (from tensorflow)
Collecting protobuf>=3.8.0 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/c1/12/7479ece04931984162698bfaa05cbb2fc23d7f6ee1ab5146cfc6ade56a31/protobuf-3.19.1-py2.py3-none-any.whl
Collecting keras-applications>=1.0.8 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/71/e3/19762fdfc62877ae9102edf6342d71b28fbfd9dea3d2f96a882ce099b03f/Keras_Applications-1.0.8-py3-none-any.whl
Collecting wheel>=0.26; python_version >= "3" (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/04/80/cad93b40262f5d09f6de82adbee452fd43cdff60830b56a74c5930f7e277/wheel-0.37.0-py2.py3-none-any.whl
Collecting six>=1.12.0 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/d9/5a/e7c31adbe875f2abbb91bd84cf2dc52d792b5a01506781dbcf25c91daf11/six-1.16.0-py2.py3-none-any.whl
Collecting wrapt>=1.11.1 (from tensorflow)
Collecting keras-preprocessing>=1.1.0 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/79/4c/7c3275a01e12ef9368a892926ab932b33bb13d55794881e3573482b378a7/Keras_Preprocessing-1.1.2-py2.py3-none-any.whl
Collecting tensorflow-estimator<2.2.0,>=2.1.0rc0 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/18/90/b77c328a1304437ab1310b463e533fa7689f4bfc41549593056d812fab8e/tensorflow_estimator-2.1.0-py2.py3-none-any.whl
Collecting google-pasta>=0.1.6 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/a3/de/c648ef6835192e6e2cc03f40b19eeda4382c49b5bafb43d88b931c4c74ac/google_pasta-0.2.0-py3-none-any.whl
Collecting absl-py>=0.7.0 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/2c/03/e3e19d3faf430ede32e41221b294e37952e06acc96781c417ac25d4a0324/absl_py-1.0.0-py3-none-any.whl
Collecting opt-einsum>=2.3.2 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/bc/19/404708a7e54ad2798907210462fd950c3442ea51acc8790f3da48d2bee8b/opt_einsum-3.3.0-py3-none-any.whl
Collecting astor>=0.6.0 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/c3/88/97eef84f48fa04fbd6750e62dcceafba6c63c81b7ac1420856c8dcc0a3f9/astor-0.8.1-py2.py3-none-any.whl
Collecting tensorboard<2.2.0,>=2.1.0 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/d9/41/bbf49b61370e4f4d245d4c6051dfb6db80cec672605c91b1652ac8cc3d38/tensorboard-2.1.1-py3-none-any.whl
Collecting termcolor>=1.1.0 (from tensorflow)
Collecting grpcio>=1.8.6 (from tensorflow)
  Using version 1.39.0 (newest version is 1.41.1, but Chaquopy prefers native wheels)
  Using cached https://chaquo.com/pypi-7.0/grpcio/grpcio-1.39.0-0-cp38-cp38-android_21_arm64_v8a.whl
Collecting chaquopy-hdf5>=1.10.2 (from h5py)
  Using cached https://chaquo.com/pypi-7.0/chaquopy-hdf5/chaquopy_hdf5-1.10.2-1-py3-none-android_21_arm64_v8a.whl
Collecting google-auth-oauthlib<0.5,>=0.4.1 (from tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/b1/0e/0636cc1448a7abc444fb1b3a63655e294e0d2d49092dc3de05241be6d43c/google_auth_oauthlib-0.4.6-py2.py3-none-any.whl
Collecting requests<3,>=2.21.0 (from tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/92/96/144f70b972a9c0eabbd4391ef93ccd49d0f2747f4f6a2a2738e99e5adc65/requests-2.26.0-py2.py3-none-any.whl
Collecting google-auth<2,>=1.6.3 (from tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/fb/7a/1b3eb54caee1b8c73c2c3645f78a382eca4805a301a30c64a078e736e446/google_auth-1.35.0-py2.py3-none-any.whl
Collecting markdown>=2.6.8 (from tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/4e/5a/501a699e2747730e03b0ed8bc59445661814ad117b31d87b8d62689d7f74/Markdown-3.3.5-py3-none-any.whl
Collecting setuptools>=41.0.0 (from tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/ed/60/15ee37d6d3385e6a432d39b5ac51f8467178ad989ba50f2b55681c1a038e/setuptools-59.1.1-py3-none-any.whl
Collecting werkzeug>=0.11.15 (from tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/1e/73/51137805d1b8d97367a8a77cae4a792af14bb7ce58fbd071af294c740cf0/Werkzeug-2.0.2-py3-none-any.whl
Collecting requests-oauthlib>=0.7.0 (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/a3/12/b92740d845ab62ea4edf04d2f4164d82532b5a0b03836d4d4e71c6f3d379/requests_oauthlib-1.3.0-py2.py3-none-any.whl
Collecting charset-normalizer~=2.0.0; python_version >= "3" (from requests<3,>=2.21.0->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/de/c8/820b1546c68efcbbe3c1b10dd925fbd84a0dda7438bc18db0ef1fa567733/charset_normalizer-2.0.7-py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests<3,>=2.21.0->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/37/45/946c02767aabb873146011e665728b680884cd8fe70dde973c640e45b775/certifi-2021.10.8-py2.py3-none-any.whl
Collecting urllib3<1.27,>=1.21.1 (from requests<3,>=2.21.0->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/af/f4/524415c0744552cce7d8bf3669af78e8a069514405ea4fcbd0cc44733744/urllib3-1.26.7-py2.py3-none-any.whl
Collecting idna<4,>=2.5; python_version >= "3" (from requests<3,>=2.21.0->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/04/a2/d918dcd22354d8958fe113e1a3630137e0fc8b44859ade3063982eacd2a4/idna-3.3-py3-none-any.whl
Collecting pyasn1-modules>=0.2.1 (from google-auth<2,>=1.6.3->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/95/de/214830a981892a3e286c3794f41ae67a4495df1108c3da8a9f62159b9a9d/pyasn1_modules-0.2.8-py2.py3-none-any.whl
Collecting rsa<5,>=3.1.4; python_version >= "3.6" (from google-auth<2,>=1.6.3->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/e9/93/0c0f002031f18b53af7a6166103c02b9c0667be528944137cc954ec921b3/rsa-4.7.2-py3-none-any.whl
Collecting cachetools<5.0,>=2.0.0 (from google-auth<2,>=1.6.3->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/ea/c1/4740af52db75e6dbdd57fc7e9478439815bbac549c1c05881be27d19a17d/cachetools-4.2.4-py3-none-any.whl
Collecting importlib-metadata>='4.4'; python_version < "3.10" (from markdown>=2.6.8->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/c4/1f/e2238896149df09953efcc53bdcc7d23597d6c53e428c30e572eda5ec6eb/importlib_metadata-4.8.2-py3-none-any.whl
Collecting oauthlib>=3.0.0 (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/e8/5d/9dd1c29e5a786525f6342f6c1d812ed2e37edc653ad297048c1668988053/oauthlib-3.1.1-py2.py3-none-any.whl
Collecting pyasn1<0.5.0,>=0.4.6 (from pyasn1-modules>=0.2.1->google-auth<2,>=1.6.3->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/62/1e/a94a8d635fa3ce4cfc7f506003548d0a2447ae76fd5ca53932970fe3053f/pyasn1-0.4.8-py2.py3-none-any.whl
Collecting zipp>=0.5 (from importlib-metadata>='4.4'; python_version < "3.10"->markdown>=2.6.8->tensorboard<2.2.0,>=2.1.0->tensorflow)
  Using cached https://files.pythonhosted.org/packages/bd/df/d4a4974a3e3957fd1c1fa3082366d7fff6e428ddb55f074bf64876f8e8ad/zipp-3.6.0-py3-none-any.whl
Installing collected packages: chaquopy-libgfortran, chaquopy-openblas, numpy, pytz, six, python-dateutil, chaquopy-libcxx, pandas, threadpoolctl, joblib, scipy, scikit-learn, gast, protobuf, chaquopy-hdf5, h5py, keras-applications, wheel, wrapt, keras-preprocessing, tensorflow-estimator, google-pasta, absl-py, opt-einsum, astor, oauthlib, charset-normalizer, certifi, urllib3, idna, requests, requests-oauthlib, pyasn1, pyasn1-modules, rsa, cachetools, setuptools, google-auth, google-auth-oauthlib, zipp, importlib-metadata, markdown, werkzeug, grpcio, tensorboard, termcolor, tensorflow
Successfully installed absl-py-1.0.0 astor-0.8.1 cachetools-4.2.4 certifi-2021.10.8 chaquopy-hdf5-1.10.2 chaquopy-libcxx-10000 chaquopy-libgfortran-4.9 chaquopy-openblas-0.2.20 charset-normalizer-2.0.7 gast-0.2.2 google-auth-1.35.0 google-auth-oauthlib-0.4.6 google-pasta-0.2.0 grpcio-1.39.0 h5py-2.10.0 idna-3.3 importlib-metadata-4.8.2 joblib-1.1.0 keras-applications-1.0.8 keras-preprocessing-1.1.2 markdown-3.3.5 numpy-1.21.4 oauthlib-3.1.1 opt-einsum-3.3.0 pandas-1.3.2 protobuf-3.19.1 pyasn1-0.4.8 pyasn1-modules-0.2.8 python-dateutil-2.8.2 pytz-2021.3 requests-2.26.0 requests-oauthlib-1.3.0 rsa-4.7.2 scikit-learn-0.24.1 scipy-1.4.1 setuptools-59.1.1 six-1.16.0 tensorboard-2.1.1 tensorflow-2.1.0 tensorflow-estimator-2.1.0 termcolor-1.1.0 threadpoolctl-3.0.0 urllib3-1.26.7 werkzeug-2.0.2 wheel-0.37.0 wrapt-1.13.3 zipp-3.6.0
Traceback (most recent call last):
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/metadata.py", line 733, in __init__
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/chaquopy/pip_install.py", line 354, in <module>
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/chaquopy/pip_install.py", line 67, in main
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/chaquopy/pip_install.py", line 136, in pip_install
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/database.py", line 211, in get_distributions
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/database.py", line 165, in _generate_cache
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/database.py", line 146, in _yield_distributions
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/metadata.py", line 743, in __init__
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/metadata.py", line 281, in __init__
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/metadata.py", line 378, in read_file
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/metadata.py", line 472, in set
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/version.py", line 704, in is_valid_matcher
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/version.py", line 104, in __init__
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/version.py", line 98, in parse_requirement
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/util.py", line 242, in parse_requirement
  File "/home/msh/Android/AndroidStudioProject/01_Official_Projects/01_Android/EdgeAI/00_on_device/00_od1/p1246_jasmy_labo.Edge_AI/EdgeAIAndroid/app/build/generated/python/bp.zip/pip/_vendor/distlib/util.py", line 211, in get_versions
SyntaxError: invalid version: '4.4'

> Task :app:generateBjitDebugPythonRequirements FAILED
> Task :app:mergeBjitDebugPythonSources UP-TO-DATE
> Task :app:javaPreCompileBjitDebug UP-TO-DATE
> Task :app:checkBjitDebugAarMetadata UP-TO-DATE
> Task :app:generateBjitDebugResValues UP-TO-DATE
> Task :app:generateBjitDebugResources UP-TO-DATE
> Task :app:mergeBjitDebugResources UP-TO-DATE
> Task :app:createBjitDebugCompatibleScreenManifests UP-TO-DATE
> Task :app:extractDeepLinksBjitDebug UP-TO-DATE
> Task :app:processBjitDebugMainManifest UP-TO-DATE
> Task :app:processBjitDebugManifest UP-TO-DATE
> Task :app:processBjitDebugManifestForPackage UP-TO-DATE
> Task :app:processBjitDebugResources UP-TO-DATE
> Task :app:mergeBjitDebugNativeDebugMetadata NO-SOURCE
> Task :app:mergeBjitDebugShaders UP-TO-DATE
> Task :app:compileBjitDebugShaders NO-SOURCE
> Task :app:generateBjitDebugAssets UP-TO-DATE
> Task :app:generateBjitDebugPythonAppAssets UP-TO-DATE
> Task :app:generateBjitDebugPythonLicenseAssets UP-TO-DATE
> Task :app:generateBjitDebugPythonMiscAssets UP-TO-DATE
> Task :app:processBjitDebugJavaRes NO-SOURCE
> Task :app:checkBjitDebugDuplicateClasses UP-TO-DATE
> Task :app:desugarBjitDebugFileDependencies UP-TO-DATE
> Task :app:mergeExtDexBjitDebug UP-TO-DATE
> Task :app:mergeLibDexBjitDebug UP-TO-DATE
> Task :app:generateBjitDebugPythonJniLibs UP-TO-DATE
> Task :app:mergeBjitDebugJniLibFolders UP-TO-DATE
> Task :app:mergeBjitDebugNativeLibs UP-TO-DATE
> Task :app:stripBjitDebugDebugSymbols UP-TO-DATE
> Task :app:validateSigningBjitDebug UP-TO-DATE

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:generateBjitDebugPythonRequirements'.
> Process 'command 'python3'' finished with non-zero exit value 1
  
  To view full details in Android Studio:
  * In version 3.6 and newer, click the 'Build: failed' caption to the left of this message.
  * In version 3.5 and older, click the 'Toggle view' button to the left of this message.
  * Then scroll up to see the full output.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 33s
27 actionable tasks: 1 executed, 26 up-to-date
```

Ubuntu version : Ubuntu 18.04.5 LTS
Android Studio version : Android Studio Arctic Fox | 2020.3.1 Patch 2 (Build #AI-203.7717.56.2031.7678000, built on August 27, 2021, Runtime version: 11.0.10+0-b96-7249189 amd64)

I've checked "Closed Issues" but didn't fulfill anything to fix my issue. 
Hope to get response from You. 
Thanks.

**Comment by mhsmith** [comment]
I can't reproduce this, but I'll keep looking into it. Meanwhile, here are a few things you could try:

* Update com.chaquo.python:gradle to the current version, 10.0.1.
* We released a new version of NumPy a few days ago, so maybe going back to the previous version will help. Try editing your build.gradle file to change "numpy" to "numpy==1.17.4".
* Or try removing "numpy" from your build.gradle file completely, and allow it to be installed automatically by the other packages.
* From the stack trace, your buildPython is version 3.6. Try updating it to use version 3.8, which will also improve your app's startup performance. If Python 3.8 isn't available in your OS's package manager, it's quite easy to compile yourself, as it uses a standard configure/make build system. Just make sure you have the OpenSSL headers installed first (e.g. "libssl-dev" on Debian).

If any of those things make a difference, even a change to the error message, please let me know.

**Comment by mhsmith** [comment]
This was actually caused by a syntax error in a requirement specification in markdown version 3.3.5. It was fixed yesterday in version 3.3.6 (https://github.com/Python-Markdown/markdown/pull/1197), so if you re-run your build now it should just work.

You can disregard my previous comment, except that updating your buildPython to version 3.8 is still a good idea because it'll significantly improve your app's startup performance, as mentioned in [the documentation](https://chaquo.com/chaquopy/doc/current/android.html#bytecode-compilation).
